pkg/parsers/operatingsystem/operatingsystem_test.go

```diff
@@ -35,24 +37,41 @@ VERSION_ID="14.04"
 HOME_URL="http://www.ubuntu.com/"
 SUPPORT_URL="http://help.ubuntu.com/"
 BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"`)
+
+		noEtcOsRelease = []byte(`NAME="Ubuntu"
+VERSION="15.04 (Vivid Vervet)"
+ID=ubuntu
+ID_LIKE=debian
+PRETTY_NAME="Ubuntu 15.04"
+VERSION_ID="15.04"
+HOME_URL="http://www.ubuntu.com/"
+SUPPORT_URL="http://help.ubuntu.com/"
+BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"`)
 	)
 
 	dir := os.TempDir()
 	etcOsRelease = filepath.Join(dir, "etcOsRelease")
+	altOsRelease = filepath.Join(dir, "altOsRelease")
```

Comment:
and here you can redefine the `[]string` to be whatever you need to be to have a mock